plugin/pkg/scheduler/api/types.go

```diff
@@ -32,6 +32,10 @@ type Policy struct {
 	Priorities []PriorityPolicy
 	// Holds the information to communicate with the extender(s)
 	ExtenderConfigs []ExtenderConfig
+	// RequiredDuringScheduling affinity is not symmetric, but there is an implicit PreferredDuringScheduling affinity rule
+	// corresponding to every RequiredDuringScheduling affinity rule.
+	// HardPodAffinitySymmetricWeight represents the weight of implicit PreferredDuringScheduling affinity rule, in the range 0-100.
+	HardPodAffinitySymmetricWeight int
```

Comment:
@k82cn If we need to move it elsewhere, we should do it in this PR IMO. This is user facing and we don't want to deprecate it again and move it.